core/tests/webdriverio_utils/ContributorDashboardAdminPage.js

```diff
@@ -238,6 +263,11 @@ var ContributorDashboardAdminPage = function() {
     await action.click('Stats row', statsListItems[0]);
     await waitFor.invisibilityOf(expandedRow, 'Expanded row still visible');
   };
+
+  this.pageToFullyInitiate = async function() {
```

Comment:
SHould we chanaeg the name to make it clear what this function is doing else it creates confusion on what it's doing? (I'm thinking of waitForLoadingMessageToDisappear or something around that?)

Reply:
Yeah, that sounds about right. updated.